src/main/java/lotto/model/BunchOfLotto.java

```diff
@@ -13,13 +14,27 @@ public BunchOfLotto(int purchasedLottoCount) {
         this.bunchOfLotto = BunchOfLottoGenerator.makeBunchOfLotto(purchasedLottoCount);
     }
 
-    public List<Lotto> getBunchOfLotto() {
-        return Collections.unmodifiableList(bunchOfLotto);
+    public BunchOfLotto() {
+        this.bunchOfLotto = new ArrayList<>();
     }
 
     public Prizes makeRewards(WinningLotto winningLotto) {
-        return new Prizes(bunchOfLotto.stream()
-                .map((lotto) -> Reward.getReward(winningLotto.makeWinningState(lotto)))
-                .collect(Collectors.toList()));
+        List<Reward> prizes = bunchOfLotto.stream()
+                .map(lotto -> Reward.getReward(winningLotto.makeWinningCount(lotto.getLottoNumbers()), winningLotto.checkBonusCount(lotto)))
```

Comment:
```suggestion
                .map(lotto -> winningLotto.makeReward(lotto)
```
위와 같이 변경할 수는 없을까요? 🤔  `Reward.getReward` 메서드를 WinningLotto의 메서드 내부에서 호출하도록 변경하는 게 더 좋을 것 같아요 😉

Reply:
파라미터를 줄일 수 있겠군요 😀